Fix analytics.js initialization timing issue and refactor

pauldowman opened the pull request:
- use analytics.ready() to call analytics.track() etc.
- reorganize files
- remove Tracker.nonreactive in a place where I think it had no effect

Fixes #47 and #58

pauldowman committed:
Fix analytics.js initialization timing issue and refactor

- use analytics.ready() to call analytics.track() etc.
- reorganize files
- remove Tracker.nonreactive in a place where I think it had no effect

benstr committed:
Update and expand example apps

benstr commented:
@pauldowman  :+1: . I also expanded the example apps to show `analytics` emit results. If it looks good to you then I can bag n' tag this PR and publish 1.0.1 to Atmosphere

pauldowman committed:
Tweaks to example apps

Show events as they're logged, not just latest, so we can be sure duplicates
and extra events aren't being logged. Also show the current identity.

pauldowman commented:
Nice! I tweaked the example apps just a little bit to show all events in order, not just the latest, to help spot duplicates and reactive re-running problems.

benstr commented:
awesome @pauldowman !
